pkg/service.go

```diff
@@ -153,6 +150,24 @@ func (s *Service) RunBeacon(ctx context.Context, client BeaconClient) error {
 	}
 }
 
+func (s *Service) waitSynced(ctx context.Context, client BeaconClient) (*SyncStatusPayloadData, error) {
+	logger := s.Log.WithField("method", "WaitSynced")
+
+	for {
+		status, err := client.SyncStatus()
+		if err != nil || !status.IsSyncing {
+			return status, err
+		}
+
+		logger.Debug("beacon clients are syncing...")
```

Comment:
Do we need that logged every minute? asking for a friend ;>

Reply:
This is only logged in the startup, until it's first synced. This way the user knows on startup when the beacon is synced and that the relay can start accepting requests

Reply:
a yea you're right, ok then